diff --git a/src/index.js b/src/index.js
index 9c2c44c6b46a33d1a1f113c8799b9fee1468e210..cc20922ea152a1f194948bfb6a8799ec43e92bcd 100644
--- a/src/index.js
+++ b/src/index.js
@@ -46,7 +46,7 @@ function waitAndRun ({ start, url, runFn, namedArguments }) {
 
   const server = execa(start, {
     shell: true,
-    stdio: ['ignore', 'inherit', 'inherit']
+    stdio: ['inherit', 'inherit', 'inherit']
   })
   let serverStopped
 
